node/src/actors/chain_manager/mod.rs

```diff
@@ -549,6 +549,15 @@ impl ChainManager {
                 return;
             }
         };
+
+        let own_pkh = match self.own_pkh {
+            Some(x) => x,
+            None => {
+                log::error!("No OwnPkh loaded in ChainManager");
```

Comment:
This is the kind of error that we should never see, isn't it? :P What happens if it is None?

Reply:
If it is None, something wrong happened. We have different ways to handle those "non optional" option fields (match, if let, unwrap...) So, we should unify in a refactor PR, but by the moment...